docs-markdown/README.md

```diff
@@ -77,6 +77,17 @@ Users of the pre-release version of the extension will notice that the authoring
 
    ![toolbar](https://github.com/Microsoft/vscode-docs-authoring/raw/master/media/image/legacy-toolbar.png)
 
+## Cleanup Scripts
+
+Cleanup functionality has been added to docs-markdown to move the experience of fixing validation closer to your development environment. Instead of having to wait until build time, you can proactively reduce the number of validation errors and warnings by running these cleanup scripts. Get started by hitting `F1` and typing `Docs: Cleanup` or hit `Alt+M` and navigate to `Cleanup...` in the quick pick menu.
+
+Cleanup scripts available:
+
+- Single-valued metadata: converts single value arrays of metadata, into inline properties and without array brackets.
+- Microsoft Links: converts http:// to https:// for microsoft docs, azure, technet, and msdn. Removes hardcoded local from url.
```

Comment:
Should this be hardcoded **locale**?

Reply:
thank you for catching that spelling mistake.